openid-sharedsignals-framework-1_0.md

```diff
@@ -853,7 +854,8 @@ stream_id
 
 > **Transmitter-Supplied**, REQUIRED. A string that uniquely identifies the stream. A
   Transmitter MUST generate a unique ID for each of its non-deleted streams
-  at the time of stream creation.
+  at the time of stream creation. Transmitters are recommended to use character set
+  described in Section 2.3 of {{rfc3986}}}} to generate the stream ID.
```

Comment:
```suggestion
  described in Section 2.3 of {{RFC3986}}}} to generate the stream ID.
```

I don't know if capitalizing this matters, but it seems like the others are capitalized.